modules/web-console-query-cli.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * web_console/web-console-query-cli.adoc
+
+[id="web-console-query-cli_{context}"]
+= Query image vulnerabilities from the CLI
+Using the `oc` command, you can display information about
+vulnerabilities detected by the Container Security Operator.
+
+.Prerequisites
+* Be running the Container Security Operator on your
+{product-title} instance
+
+.Procedure
+To query for detected container image vulnerabilities, type:
+
+```
```

Comment:
Same thing I mentioned above, this isn't the correct syntax for code blocks. Here and another instance below in this file.

Reply:
Done.